miqsel/config.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+import click
+import yaml
+
+
+class Configuration(object):
+    """read write conf yaml files"""
+
+    def __init__(self, conf_file=None):
+        self.conf_file = (
+            conf_file if conf_file else os.path.join(os.path.dirname(__file__), "conf.yaml")
+        )
+
+    def read(self):
+        with open(self.conf_file, "r") as ymlfile:
+            return yaml.load(ymlfile)
+
```

Comment:
@digitronik 
- I have run this code locally but I am getting below warning: 
```
(miq) [ghubale@localhost testing]$ miqsel appliance
/home/ghubale/testing/miq/lib64/python3.6/site-packages/miqsel-1.4-py3.6.egg/miqsel/config.py:17: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  return yaml.load(ymlfile)
```
- I went through the link: https://msg.pyyaml.org/load 
- I think, we can avoid this warning by specifying ```Loader=``` argument:
```yaml.load(ymlfile, Loader=yaml.FullLoader)```

Reply:
I am not able to reproduce this... please provide me details like which python you are using and full log.

Reply:
Q: @digitronik May be your environment have enough resources(pkgs installed) and because of that you are not getting errors .?

Reply:
Its `PyYAML` deprecation... this package going to deprecated we will move to `ruamel.yaml` :+1: For now ignore this.